dronekit_sitl/__init__.py

```diff
@@ -268,7 +347,23 @@ def cleanup_sim():
         # else:
         #     sitl = Popen(sitl_args, stdout=PIPE, stderr=PIPE)
 
-        p = Popen([self.path] + args, cwd=wd, shell=sys.platform == 'win32', stdout=PIPE, stderr=PIPE)
+        popen_args = []
+        if self.gdb:
```

Comment:
is there a way to write an integration test with `gdb`? I'm interested in testing the `gdb` support, lets say we launch sitl and use a known breakpoint, can we kill the session then resume operations and catch that here?